src/main/java/com/charan/parkingsystem/strategy/BikeStrategy.java

```diff
@@ -0,0 +1,27 @@
+package com.charan.parkingsystem.strategy;
+
+import com.charan.parkingsystem.parking.ParkingFloor;
+import com.charan.parkingsystem.parking.ParkingSlot;
+import com.charan.parkingsystem.parking.Ticket;
+import com.charan.parkingsystem.vehicle.Vehicle;
+
+import java.util.List;
+
+public class BikeStrategy implements ParkStrategy {
```

Comment:
Using strategy pattern here is unnecessary as parking behaviour is almost same for all vehicles. Check the design pattern once see the example usecases of it.

Reply:
And `park` method code seems to be same for all vehicles. Refactor it.